### ISSUE TITLE
Dead link in docs rllib-dev.rst

### ISSUE THREAD

**Issue by jacekplocharczyk** [ray-project/ray#6402]
There is a dead link in the docs https://github.com/ray-project/ray/blob/ray-0.4.0/doc/source/rllib-dev.rst

> Define an algorithm-specific Policy evaluator class (the core of the algorithm). Evaluators encapsulate framework-specific components such as the policy and loss functions. For an example, see **[the simple policy gradient evaluator example.](https://github.com/ray-project/ray/blob/master/python/ray/rllib/pg/pg_evaluator.py)**

The same here:
> To help with implementation, RLlib provides common action distributions, preprocessors, and neural network models, found in **[catalog.py](https://github.com/ray-project/ray/blob/master/python/ray/rllib/models/catalog.py),** which are shared by all algorithms. Note that most of these utilities are currently Tensorflow specific.

**Comment by hhbyyh** [comment]
The doc is for Ray 0.4.0. Is it the version you're using? The latest doc https://ray.readthedocs.io/en/latest/rllib.html

**Comment by jacekplocharczyk** [comment]
I was talking about this file from repo `doc/source/rllib-dev.rst` but I see that it wasn't updated since 2018 so nvm. I thought that it's the same version as https://ray.readthedocs.io/en/latest/rllib.html.

Thanks
